fawkes/algorithms/algo.py

```diff
@@ -67,11 +67,14 @@ def run_algo():
         constants.APP_CONFIG_FILE.format(file_name=constants.APP_CONFIG_FILE_NAME)
     )
     for app_config_file in app_configs:
+        app_config_json = utils.open_json(
+            app_config_file
+        )
+        utils.validate_app_config_schema(app_config_json)
```

Comment:
This should not be a function in utils. Validation of AppConfig schema is something internal to AppConfig class.

Pls move it there.

Reply:
It will also save the repetition of code everywhere.

Whenever an app. config is initialised, schema validation should be done.

Reply:
This change is done.